README.md

```diff
@@ -15,7 +15,18 @@ setup a Yocto Project build environment for you!
 
 * **default.xml** - External releasable components. Used for release builds.
 
-## Getting Started
+## Reproducing image measurements
+
+**Make sure you are checked out on the correct commit (in this repo).**
+
+Building images is as simple as installing docker, and running `make image-<>`. For list of images run `make help`.  
+Make sure you have plenty of disk space available (200 GBs), and that your session will not time out as a fresh build can take a couple of hours on slow hardware.  
+
+Measurements will be available in <> once the image build process is finished.  
```

Comment:
measurements will be available in the measurements directory upon successful build and execution of the measurement generation script